src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -374,7 +374,9 @@ private static void CreateMethod(string propIdentifier, string propType, CodeCla
                 Kind = CodeParameterKind.Path,
                 SerializationName = parameter.Name.Equals(codeName) ? default : parameter.Name.SanitizeParameterNameForUrlTemplate(),
             };
-            mParameter.Type = GetPrimitiveType(parameter.Schema);
+            mParameter.Type = GetPrimitiveType(parameter.Schema ?? parameter.Content.Values.FirstOrDefault()?.Schema);
+            mParameter.Type.CollectionKind = parameter.Schema.IsArray() ? CodeType.CodeTypeCollectionKind.Array : default;
```

Comment:
QQ. If `parameter.Schema` can be null above, can we still call `parameter.Schema.IsArray()` safely without an exception throwing?

Reply:
Resolving. 
Looks like the extension does perform the null check here. 
https://github.com/microsoft/kiota/blob/2d71b6825420379ba71a2369b9caefa77d0584db/src/Kiota.Builder/Extensions/OpenApiSchemaExtensions.cs#L58